src/components/Table/TableWithActions.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import { Table } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
 import style from './Table.module.css';
 
 export function TableWithActions({ titles, items, linkPath, action }) {
   return (
-    <table className={style.table}>
+    <Table striped bordered hover className={style.table}>
```

Comment:
pass this variable as params with default values to be able to rewrite them

Reply:
fixed
```
export function Table({ title, items, striped = true, bordered = true, hover = true }) {
  return (
    <BootstrapTable striped={striped} bordered={bordered} hover={hover}>
      <TableHead items={title} />
      {items.map((item) => item)}
    </BootstrapTable>
  );
}
```